co2calculator/calculate.py

```diff
@@ -305,28 +247,8 @@ def calc_co2_train(
         warnings.warn(
```

Comment:
Just spotted a small mistake in L 243:
should be "Train" not "Car": 
`f"Train fuel type was not provided. Using default value: '{fuel_type}'"`